api/src/main/java/com/datastrato/gravitino/rel/Column.java

```diff
@@ -23,6 +24,7 @@
 public interface Column {
 
   Expression DEFAULT_VALUE_NOT_SET = () -> Expression.EMPTY_EXPRESSION;
+  Expression DEFAULT_VALUE_OF_CURRENT_TIMESTAMP = FunctionExpression.of("current_timestamp");
```

Comment:
Add Javadocs for this variable and above.

Reply:
fixed